content/en/blog/2022/debug-otel-with-otel/index.md

```diff
@@ -205,35 +205,35 @@ Once again we ran `docker compose up`[^1] to bring up our sample app and we send
 some request with `curl localhost:8080` to the frontend application.
 
 In Jaeger we still see that the traces are disconnected. However, when we look
-into one of those traces, we can see the collected request headers from nginx to
+into one of those traces, we can see the collected request headers from NGINX to
 backend:
 
 ![A screenshot of the Jaeger UI showing that http.request.header.traceparent has multiple entries.](jaeger-with-request-headers.png)
 
 There it is! The trace headers (`baggage`, `traceparent`, `tracestate`) are send
-as multiple header fields: the nginx module added the value of each of those
+as multiple header fields: the module for NGINX added the value of each of those
```

Comment:
The original "the NGINX module added ..." seemed ok to me, why change the word ordering (here and later in the doc)? To me "the NGINX module" can be interpreted as a shorthand for "the module for NGINX". Besides, that's how they write it in https://opentelemetry.io/blog/2022/instrument-nginx/, which is referred to at the top of this article. But if you feel that this is more correct, I'm ok with the change.

Reply:
let me revert that